taskcluster/ci/beetmover/kind.yml

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: mozillavpn_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - mozillavpn_taskgraph.transforms.multi_dep:transforms
+    - mozillavpn_taskgraph.transforms.beetmover:transforms
+    - taskgraph.transforms.task:transforms
+
+group-by: build-type
+
+only-for-build-types:
+    - macos/opt
+
+kind-dependencies:
+    - signing
+
+primary-dependency: signing
+
+job-template:
+    worker-type: beetmover
+    run-on-tasks-for: [action, github-release]
```

Comment:
I'm not sure `action` is a valid tasks_for. Instead the action we'll use to trigger this will just use a `target_tasks_method` that explicitly selects this task. It won't even need to look at the `run-on-tasks-for` property.